e2e/babel-plugin-jest-hoist/.babelrc

```diff
@@ -1,12 +0,0 @@
-{
-  "presets": [
-    "es2015",
-    {
-      "plugins": [
-        "jest-hoist",
-        "transform-flow-strip-types"
-      ]
-    }
-  ],
-  "retainLines": true
-}
```

Comment:
why was this removed?

Reply:
In the issue I was having here: #7466. i was able to resolve it by removing the file. But I accidentally added this to the last commit it wasn't supposed to be pushed. I meant to ask about adding this separately since it seemed to have resolved the issue for me.

Reply:
I'm a bit confused as to where to put the newline, I copied the contents back from the original file over to this file from github. 😕

Reply:
Scratch that! I found it. :)